src/core.c/ThreadPoolScheduler.pm6

```diff
@@ -781,8 +781,8 @@ my class ThreadPoolScheduler does Scheduler {
     }
 
     submethod BUILD(
-        Int :$!initial_threads = 0,
-        Int :$!max_threads = (%*ENV<RAKUDO_MAX_THREADS> // 64).Int
+      Int() :$!initial_threads = 0,
+      Int() :$!max_threads = %*ENV<RAKUDO_MAX_THREADS> // Kernel.cpu-cores * 8;
```

Comment:
I wonder if this should maybe be `Kernel.cpu-cores * 8 max 64`; there may be programs out there relying on the current upper bound, that would end up with a lower one here, and could start to deadlock as a result.

Reply:
@jnthn I was deadlocking on insufficient number of threads, until realized how things work. But too many?

Anyway, this parameter is an implementation detail. No one must rely on this number.

Reply:
>  But too many?

`max` would mean "no less than 64", e.g. retaining the current number, so no existing program will run up against a new insufficiency.

> Anyway, this parameter is an implementation detail. No one must rely on this number.

I agree, but what people should do and what people do do is a different matter. It costs us almost nothing to know we won't make something newly deadlock, and is cheaper than dealing with it if we do.

Reply:
@jnthn Sorry, it's my eternal problem with mixing `max`/`min` outcomes, akin to some people mixing left/right.

So, my comment is only relevant to when increasing the number would cause problems.

Reply:
Aha. :-) Yes, increasing the number is much less likely to cause trouble (and if it does, it's higher resource usage trouble, not deadlocked program trouble).